src/ListComponent.js

```diff
@@ -94,6 +112,7 @@ const ListComponent = (props) => {
                                                     onClick={() => editTask(id)}> 
                                                     Edit 
                                                 </Button> 
+                                              <input type = "checkbox" value={done} onChange={()=>handleChange("done")} /> done
```

Comment:
So here I would think onChange would just cycle the item.done value...unsure on exact syntax but something like `item.done = ! item.done. And I think value would be item.done in this case too?